client/state/followers/thunks/remove-follower.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:followers-actions' );
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+import { requestRemoveFollower, successRemoveFollower, failRemoveFollower } from '../actions';
+
+/**
+ * Request removal of a follower for a given site.
+ *
+ * @param {number} siteId site identifier
+ * @param {object} follower contains the follower object
+ */
+function removeFollower( siteId, follower ) {
```

Comment:
Another good candidate for porting to the data layer instead of creating a new thunk.

Reply:
done via d056358